File: evaluator/src/main/java/org/arnhold/evaluation/constraints/SemanticServiceImpl.java
package org.arnhold.evaluation.constraints;

import org.apache.jena.rdf.model.Model;
import org.apache.jena.riot.RDFDataMgr;
import org.springframework.stereotype.Component;

import java.io.File;

@Component
public class SemanticServiceImpl implements SemanticService {

    @Override
    public Model loadModelFromFile(File file) {
        return RDFDataMgr.loadModel(file.getAbsolutePath());
    }
}

File: evaluator/src/main/kotlin/org/arnhold/evaluator/method/constraints/SemanticServiceImpl.kt
package org.arnhold.evaluator.method.constraints

import org.apache.jena.rdf.model.Model
import org.springframework.stereotype.Component
import org.apache.jena.riot.RDFDataMgr
import java.io.File

@Component
class SemanticServiceImpl : SemanticService {
    override fun loadModelFromFile(file: File): Model {
        return RDFDataMgr.loadModel(file.absolutePath)
    }
}
